realm/realm-jni/src/util.hpp

```diff
@@ -578,6 +578,7 @@ class JniLongArray {
     ~JniLongArray()
     {
         m_env->ReleaseLongArrayElements(m_javaArray, m_array, m_releaseMode);
+        m_env->DeleteLocalRef(m_javaArray);
```

Comment:
We should add similar to all our array classes in `util.hpp` to prevent memory leaks.